src/scanner/images/index.ts

```diff
@@ -119,10 +119,13 @@ export async function scanImages(
   for (const { imageName, fileSystemPath, imageWithDigest } of images) {
     try {
       const archivePath = `docker-archive:${fileSystemPath}`;
+      const imageNameAndDigest: string | undefined =
+        imageWithDigest ?? undefined;
```

Comment:
I think the `??` operator will preserve the empty string, which is probably not what we want. Perhaps we could use `imageWithDigest | undefined`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
> ... is a logical operator that returns its right-hand side operand when its left-hand side operand is [null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/null) or [undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined)

Reply:
Oops, I missed that. I reversed the order, so now if `''`, `undefined` would be returned first.